entries/s/skrill.com.json

```diff
@@ -16,7 +15,118 @@
       "payments"
     ],
     "regions": [
-      "-jp"
+	  "-af",
+	  "-ax",
+	  "-as",
+	  "-ao",
+	  "-ai",
+	  "-aq",
+	  "-ag",
+	  "-bb",
+	  "-bz",
+	  "-bj",
+	  "-bt",
+	  "-bq",
+	  "-bv",
+	  "-io",
+	  "-bf",
+	  "-bi",
+	  "-cv",
+	  "-cf",
+	  "-td",
+	  "-cc",
+	  "-km",
+	  "-cd",
+	  "-cg",
+	  "-ck",
+	  "-cu",
+	  "-cw",
+	  "-dj",
+	  "-gq",
+	  "-er",
+	  "-fo",
+	  "-fj",
+	  "-gf",
+	  "-pf",
+	  "-tf",
+	  "-ga",
+	  "-gm",
+	  "-gl",
+	  "-gd",
+	  "-gp",
+	  "-gn",
+	  "-gw",
+	  "-gy",
+	  "-ht",
+	  "-hm",
+	  "-va",
+	  "-ir",
+	  "-iq",
+	  "-ci",
+	  "-jp",
+	  "-ki",
+	  "-kg",
+	  "-la",
+	  "-lb",
+	  "-lr",
+	  "-ly",
+	  "-mo",
+	  "-ml",
+	  "-mh",
+	  "-mq",
+	  "-mr",
+	  "-yt",
+	  "-fm",
+	  "-ms",
+	  "-mm",
+	  "-nr",
+	  "-nc",
+	  "-ne",
+	  "-nu",
+	  "-nf",
+	  "-kp",
+	  "-mp",
+	  "-pw",
+	  "-ps",
+	  "-pg",
+	  "-re",
+	  "-pn",
+	  "-rw",
+	  "-bl",
+	  "-sh",
+	  "-kn",
+	  "-lc",
+	  "-mf",
+	  "-pm",
+	  "-vc",
+	  "-st",
+	  "-sc",
+	  "-sl",
+	  "-sx",
+	  "-sb",
+	  "-so",
+	  "-gs",
+	  "-sd",
+	  "-ss",
+	  "-sr",
+	  "-sj",
+	  "-sy",
+	  "-tj",
+	  "-tl",
+	  "-tg",
+	  "-tk",
+	  "-to",
+	  "-tr",
+	  "-tm",
+	  "-tc",
+	  "-tv",
+	  "-vu",
+	  "-ve",
+	  "-wf",
+	  "-eh",
+	  "-ws",
+	  "-ye",
+	  "-zw"
```

Comment:
two more spaces would be preferred by our editorconfig.
Also, I think you accidentally removed the blank line at the end of the file :wink: 

Depending on what editor you're using, there might be an [editorconfig](https://editorconfig.org/) plugin available.